tests/gen_vectors/gen_uint128_intrinsics_vectors.py

```diff
@@ -0,0 +1,66 @@
+import os
```

Comment:
can you make sure this test generates boundary conditions, e.g. UINT_MAX, UINT_MAX, along with 0, UINT_MAX, and UINTMAX, 0?